assets/src/edit-story/components/library/panes/text/textSets/index.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import getTextSets from './getTextSets';
+
+export default async function () {
+  const textSetLibrary = await getTextSets();
+
+  return Object.values(textSetLibrary).reduce((textSets, set) => {
+    return [...textSets, ...set];
+  }, []);
```

Comment:
For now, just collapsing all the text sets into one big array.  In the future we'll be applying filtering to only return what we need.

Reply:
Isn't this what `array.flat()` does?